### PR TITLE
ToDo List code added

### PR DESCRIPTION
**Pull request by kunal232i**
## Description
Completed issue #104 
ToDO List code added to Level0 with a Readme file.

## Type of change

- [x] Add a new project

- [x] Modified or enhanced the existing projects

- [x] Improve the readme of any project

## Domain of Contribution

- [x] Project using only Solidity

- [x] Project Test on the Remix IDE

## Checklist ✓

- [x] l have performed a self-review of my own code or work.

images

### CODE REVIEW

File: Level0/ToDo List/Readme.md [hunk 1]
```diff
@@ -20,22 +20,23 @@ Type the file name and enter the respective code into it.
...
 
-Click “>” before your contract you will see a button Calculator below as the contract has addition,multiplication,division and substractiono function that returns the ans of the number in getans.
+Click “>” before your contract you will see a button todolist below as the contract has addTodo, completeTodo, removeTodo, uncompleteTodo and getAllTodos function that returns the all todo list in getAllTodos.
 
-Enter a two numbers and click the function whatever you want button and you will get the output in getans:
+Enter Whatever tasks you want, click the functions that correspond to your tasks, and you'll get the following output in getAllTodos:
+
+![Deployed](https://user-images.githubusercontent.com/81668653/193433422-fc246b10-e016-48f2-93d1-930363584f8b.png)
 
-![image](https://user-images.githubusercontent.com/81668653/193393870-0e8e589f-191b-400c-b696-4fe559d6cbf2.png)
```
Comment (Vikash-8090-Yadav) [inline Level0/ToDo List/Readme.md:41]: after step 4 please  explain function with their use .

Comment (kunal232i) [reply Level0/ToDo List/Readme.md:41]: > after step 4 please explain function with their use .

ohk 👍

Comment (kunal232i) [reply Level0/ToDo List/Readme.md:48]: > after step 4 please explain function with their use .

Hello there, added functionalities in step 4according to [this](https://github.com/Vikash-8090-Yadav/Solidity-Pathshala/pull/114#discussion_r985179408) have a look at it [Readme](https://github.com/kunal232i/Solidity-Pathshala/tree/main/Level0/ToDo%20List#step-4). 😄👍

**Comment by Vikash-8090-Yadav** [comment]
please change the pr heading and give some legit heading ,

**Comment by Vikash-8090-Yadav** [comment]
Now all set to  merge .  Ping me  once its when completed @kunal232i

**Comment by kunal232i** [comment]
> Now all set to merge . Ping me once its when completed @kunal232i

Hey, I am done with PR change now you can merge it.👍

**Comment by kunal232i** [comment]
> > Now all set to merge . Ping me once its when completed @kunal232i
> 
> Hey, I am done with PR change now you can merge it.👍

**Comment by Vikash-8090-Yadav** [comment]
Great @kunal232i  . Hope  to see more contribution from your side . 
Thanks for the contribution  .
Happy contribution

**Pull request merged**